source/gui/settingsDialogs.py

```diff
@@ -792,7 +794,8 @@ def onSave(self):
 		config.conf["general"]["askToExit"]=self.askToExitCheckBox.IsChecked()
 		config.conf["general"]["playStartAndExitSounds"]=self.playStartAndExitSoundsCheckBox.IsChecked()
 		logLevel=self.LOG_LEVELS[self.logLevelList.GetSelection()][0]
-		config.conf["general"]["loggingLevel"]=logging.getLevelName(logLevel)
+		if not logHandler.isLogLevelForced:
+			config.conf["general"]["loggingLevel"] = logging.getLevelName(logLevel)
 		logHandler.setLogLevelFromConfig()

```

Comment:
I think this has to belong to the if clause.

```suggestion
			logHandler.setLogLevelFromConfig()
```

Reply:
It indeed can, with a minor performance improvement. as `logHandler.setLogLevelFromConfig()` begins by checking for the same condition.